lib/Runtime/Library/JavascriptArray.h

```diff
@@ -1137,6 +1137,8 @@ namespace Js
         uint32 GetNextIndex(uint32 index) const;
         BOOL DirectGetItemAt(uint32 index, int* outVal);
 
+        virtual PropertyQueryFlags GetPropertyQuery(Var originalInstance, PropertyId propertyId, Var* value, PropertyValueInfo* info, ScriptContext* requestContext) override;
```

Comment:
Looks like we also need `#if ENABLE_COPYONACCESS_ARRAY` for these method definitions.